cdi_test/tests/run-cdi.sh

```diff
@@ -115,13 +115,13 @@ if [[ ${RUN_MINIMAL} -eq 1 ]]; then
     set +e
     echo "==> Running minimal cdi_test"
 
-    cdi_execute_cmd ${INSTANCE_IPS[0]} \
+    timout --kill-after=5m cdi_execute_cmd ${INSTANCE_IPS[0]} \
```

Comment:
Minor: let's define variables like `cdi_test_timeout` and refer it in both server and client to force them consistent